examples/Components/linearsolver/Eigen3-SVD.scn

```diff
@@ -1,14 +1,5 @@
 <?xml version="1.0"?>
 <Node name="root" dt="0.02" gravity="0 -10 0">
-    <RequiredPlugin name="Sofa.Component.Constraint.Projective"/> <!-- Needed to use components [FixedConstraint] -->
```

Comment:
strange that no plugin are needed for this one

Reply:
Yes, probably an unfortunate Ctr + Z... Thanks for spotting it!